### PR TITLE
Handle retro compat with Google SDK < 20

### PR DESCRIPTION
**Pull request by TurpIF**
JIRA: DPP-3348

### CODE REVIEW

File: buildSrc/src/main/java/Deps.kt [hunk 1]
```diff
@@ -1,21 +1,3 @@
-/*
```
Comment (vguerci) [inline buildSrc/src/main/java/Deps.kt:1]: NIT: Any particular reason removing header?

**Comment by TurpIF** [comment]
Coverage is low because the new dedicated module is testing the publisher-sdk module. And coverage across module is not yet supported by the jacoco setup. See https://jira.criteois.com/browse/EE-1370

**Review by vguerci — APPROVED** [review]

File: buildSrc/src/main/java/Deps.kt (hunk 1 above)
Comment (TurpIF) [reply buildSrc/src/main/java/Deps.kt:1]: Oh, at first sight I thought that my IDE removed this without my consent.
By in fact, you can see that, before, there was 2 headers and that my IDE automatically cleaned one. Smart IDE 😄

Comment (vguerci) [reply buildSrc/src/main/java/Deps.kt:1]: Smart enough to determine copyright isn't really relevant on deps ¯\\\_(ツ)\_/¯

**Pull request merged**